From 9845835a37a1834d50a944aa5519f668e6ca5915 Mon Sep 17 00:00:00 2001
From: vinothneevee <vinoth@neeveetech.com>
Date: Tue, 5 Mar 2024 14:14:16 +0530
Subject: [PATCH] debug ptint

---
 board/toradex/colibri_imx7/colibri_imx7.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/board/toradex/colibri_imx7/colibri_imx7.c b/board/toradex/colibri_imx7/colibri_imx7.c
index 486299b5e9..f255bef526 100644
--- a/board/toradex/colibri_imx7/colibri_imx7.c
+++ b/board/toradex/colibri_imx7/colibri_imx7.c
@@ -53,6 +53,8 @@ DECLARE_GLOBAL_DATA_PTR;
 
 #define USB_CDET_GPIO	IMX_GPIO_NR(7, 14)
 
+#pragma message(" uart patch debug print ")
+
 int dram_init(void)
 {
 	gd->ram_size = get_ram_size((void *)PHYS_SDRAM, imx_ddr_size());
-- 
2.25.1

